.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea/
 *.iml
+/dotnet/html-formatter/.vs
```

Comment:
Please move to the . gitignore in dotnet.

Reply:
Fixed